src/types/onyx/PlaidData.ts

```diff
@@ -15,6 +15,7 @@ type PlaidData = {
     bankAccounts?: PlaidBankAccount[];
 
     isLoading?: boolean;
+    error?: string;
```

Comment:
I think this is some leftover too. I have asked to get a confirmation on this one

Reply:
Didn't get a response yet. Tagged @chiragsalian in the corresponding discussion

Reply:
I don't immediately see error key being used in the backend. Do we know which flow might return an error key, because if we don't then i think we should remove this.

Reply:
Let's remove it then cc @fabioh8010